setup.py

```diff
@@ -105,6 +105,7 @@
         "pydantic~=1.10,>=1.10.2",
         "email_validator~=1.2",
         "requests~=2.28",
+        "pyyaml>=5.4, <7",  # Keep at pyyaml 5.4 for aiida-core support
```

Comment:
Did you pin down which part of the non-server stuff needs pyyaml? I assume it was just for reading the config files in yaml format previously